src/createPersistor.js

```diff
@@ -12,6 +12,7 @@ export default function createPersistor (store, config) {
   const transforms = config.transforms || []
   const debounce = config.debounce || false
   const keyPrefix = config.keyPrefix !== undefined ? config.keyPrefix : KEY_PREFIX
+  const createFragmentedKey = config._createFragmentedKey || defaultCreateFragmentedKey
```

Comment:
if u pass some option in the config, it for sure shouldnt be named like this `_createFragmentedKey` which indicates something private